### DISCUSSION TITLE
Does not match checksums on maven and 1.4.1 bin archive

### DISCUSSION THREAD

**Discussion by yazynin** [apache/ozone#7560] [General]
Is it ok what checksum on maven don't match?
for example:
```
curl -l "https://dlcdn.apache.org/ozone/1.4.1/ozone-1.4.1.tar.gz" -o ozone-1.4.1.tar.gz

curl "https://repo1.maven.org/maven2/org/apache/ozone/ozone-filesystem-hadoop3/1.4.1/ozone-filesystem-hadoop3-1.4.1.jar" -o ozone-filesystem-hadoop3-1.4.1.jar

➜  test md5sum ozone-1.4.1/ozone-1.4.1/share/ozone/lib/ozone-filesystem-hadoop3-1.4.1.jar
f8d13127beb323743d47c38d1b256eac  ozone-1.4.1/ozone-1.4.1/share/ozone/lib/ozone-filesystem-hadoop3-1.4.1.jar

➜  test md5sum ozone-filesystem-hadoop3-1.4.1.jar
e4098ba0470e2e527968a84eeaf17bda  ozone-filesystem-hadoop3-1.4.1.jar

➜  test ls -la ozone-1.4.1/ozone-1.4.1/share/ozone/lib/ozone-filesystem-hadoop3-1.4.1.jar
-rw-r--r--  1 nikolay.yazynin  staff  71428708 14 ноя 12:19 ozone-1.4.1/ozone-1.4.1/share/ozone/lib/ozone-filesystem-hadoop3-1.4.1.jar
➜  test ls -la ozone-filesystem-hadoop3-1.4.1.jar
-rw-r--r--  1 nikolay.yazynin  staff  71428461 11 дек 14:14 ozone-filesystem-hadoop3-1.4.1.jar

```

**Comment by adoroszlai** [comment]
Thanks @yazynin for reporting this.  It looks like Ozone was built for two different steps in the release process.  We need to update Ozone's Release Manager guide to make sure the same jars included in the tarball are uploaded to Maven.

You can examine the difference between the two jars by using [diffoscope](https://diffoscope.org/).

**Comment by jojochuang** [comment]
We will be using the updated release process for Ozone 2.0 and onwards: https://ozone-site-v2.staged.apache.org/docs/developer-guide/project/release-guide/
so you should expect to see matching jars moving forward. Thanks for reporting it. I'll close this Discussion once we complete 2.0 release work.
